src/modules/Modal/Modal.js

```diff
@@ -52,6 +52,12 @@ class Modal extends Component {
     /** A modal can reduce its complexity */
     basic: PropTypes.bool,
 
+    /** Closes the modal if Escape is pressed. Defaults to true */
```

Comment:
Let's leave the default value out of the descriptions as it is documented automatically from `defaultProps`:

![image](https://cloud.githubusercontent.com/assets/5067638/18017047/62c35e9e-6b84-11e6-962e-1cc0bd2a736d.png)